source/solvers/navier_stokes_assemblers.cc

```diff
@@ -1861,6 +1861,276 @@ WeakDirichletBoundaryCondition<dim>::assemble_rhs(
+                                            beta_tangent *
+                                            (scratch_data
+                                               .face_phi_u[f][q][j][comp_i] -
+                                             scratch_data
+                                                 .face_normal[f][q][comp_i] *
+                                               (scratch_data
+                                                  .face_normal[f][q][comp_i] *
+                                                scratch_data
+                                                  .face_phi_u[f][q][j]
+                                                             [comp_i])) *
+                                            scratch_data
+                                              .face_phi_u[f][q][i][comp_i] *
+                                            JxW;
+
+                                          /* We don't use these terms has its
+                                           * seems they don't affect the
+                                           * solution for the normal component
+                                           */
+                                          /*
+                                                                                    double grad_phi_terms =
+                                                                                      scratch_data.face_normal[f][q][comp_i]*((viscosity *
+                                                                                        scratch_data
+                                                                                          .face_phi_u[f][q][j]) *
+                                                                                       (scratch_data
+                                                                                          .face_grad_phi_u[f][q][i] *
+                                                                                        scratch_data.face_normal[f][q])) *
+                                                                                      JxW;
+                                                                                    double surface_stress_term =
+                                                                                      scratch_data.face_normal[f][q][comp_i]* (viscosity *
+                                                                                       scratch_data
+                                                                                         .face_grad_phi_u[f][q][j] *
+                                                                                       scratch_data.face_normal[f][q]) *
+                                                                                      scratch_data.face_phi_u[f][q][i] *
+                                                                                      JxW;
+                                          */
```

Comment:
should this part be erased? Or do you want to keep it for later?

Reply:
It should be kept there for now.